docs/bigquery_sink.md

```diff
@@ -39,35 +39,48 @@ Require configuration for:
 Accept optional configuration for:
 
  * The fallback output PubSub topic for messages with no route
- * The output mode for BigQuery, default to `STREAMING_INSERTS`
+ * The output mode for BigQuery, default to `mixed`
```

Comment:
The case-ness of `STREAMING_INSERTS` vs. `mixed` is interesting and I am guessing it is because `mixed` is a custom notion of ours whereas `STREAMING_INSERTS` maps to a specific beam or dataflow concept.

Reply:
Yes, the all-caps version is BQ's whereas we provide an extra layer of abstraction and user lowercase. I will change the FILE_LOADS reference below to also use our lowercase variant.